src/app/project/controllers/collect-records.ctrl.js

```diff
@@ -404,13 +447,30 @@ angular.module('app.project').controller('CollectRecordsCtrl', [
       });
     };
 
+    $scope.tableControl.getFilteredRecordsCount = function() {
+      return (
+        $scope.tableControl.records &&
+        `${$scope.tableControl.records.length}/${collectRecordsCount}`
+      );
+    };
+
+    $scope.tableControl.hideFilteredCount = function() {
+      return (
+        $scope.tableControl.records &&
+        $scope.tableControl.records.length === collectRecordsCount
+      );
+    };
+
     promises = [
       authService.getCurrentUser(),
       offlineservice.CollectRecordsTable(project_id)
     ];
 
     $q.all(promises).then(function(output) {
```

Comment:
If these promises don't resolve, `collectRecordsCount` doesn't get set -- maybe it should be given a default on declaration